summarize/Readme.md

```diff
@@ -2,17 +2,48 @@
 
 Summarize is a tool to produce a human readable summary of `measureme` profiling data.
 
-## Example
+## Installing summarize
+
+In order to use this tool you will first want to build it. To do this follow the steps
```

Comment:
```suggestion
To use this tool you will first want to build it. To do this follow the steps
```

Reply:
Ugh, I pressed enter too soon on the commit message in the github UI and now I can't change it.

Reply:
Okay I fixed it.